addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -363,12 +373,28 @@ a.o_underline {
             height: 100%;
             cursor: ew-resize;
         }
+    }
+}
 
-        .ace_scroller.o_error::after {
```

Comment:
Maybe you could keep this one (with same color and same position) so when scrolling down, the visual indication of an error is not lost.
Not easy to explain so here is a step to reproduce:
- Open a file in HTML editor which has scroll
- Remove a closing tag or whatever to produce an error on top of the tile
- Scroll down
Before: the red line was kept
Now: It is gone, only appearing back when the scroll go back on the error

If not clear enough, screenshots :D (error is line 10 or smth)

Before:

![image](https://user-images.githubusercontent.com/30048408/143279027-e3c464f2-6ac4-4731-a147-c13cd24ec276.png)

After:
![image](https://user-images.githubusercontent.com/30048408/143279107-bd831be1-f9ad-47e9-846d-65d296d10479.png)

Reply:
Good catch, solved as discussed on Discord (showing the error icon at first or last line if the user has to scroll)